console/src/components/Services/Data/TableBrowseRows/ViewRows.js

```diff
@@ -950,6 +951,7 @@ const ViewRows = ({
           persistColumnOrderChange(curTableName, currentSchema, reorderData)
         }
         defaultReorders={columnsOrder}
+        showPagination={!shouldHidePagination}
```

Comment:
Can we use the same prop name throughout the tree, to avoid confusion? :see_no_evil: